README.md

```diff
@@ -6,6 +6,16 @@
 
 The Reconciler is a central system to reconcile Kyma clusters.
 
+## Locally start reconciler service
+
+1. Build docker image 
+
+`docker build -f Dockerfile -t reconciler:v1 .`
+
+2. Run docker container
+
+`docker run --name reconciler -it -p 8080:8080 reconciler:v1 reconciler service start`
```

Comment:
```suggestion
```docker run --name reconciler -it -p 8080:8080 reconciler:v1 reconciler service start```
```